velocidi-sdk/src/test/kotlin/com/velocidi/VelocidiTest.kt

```diff
@@ -78,13 +75,16 @@ class VelocidiTest {
 
         val context: Context = ApplicationProvider.getApplicationContext()
 
-        Velocidi.instance = VelocidiMockSync(config, context)
+        Velocidi.instance = Velocidi(config, context)
 
-        Velocidi.getInstance().match("provider1", listOf(UserId("eml", "mail@example.com")))
+        Velocidi.getInstance().match(
+            "provider1",
+            listOf(UserId("123"), UserId("mail@example.com", "eml"))
```

Comment:
Nitpick: can we remove literal email examples? This example in particular is not consistent with the id _type_ and I think we should avoid exemplifying plain emails... Can we use a proper sha256 example here and in other examples?